pkg/function/create_volume_from_snapshot.go

```diff
@@ -115,7 +116,9 @@ func createVolumeFromSnapshot(ctx context.Context, cli kubernetes.Interface, nam
 		if err != nil {
 			return nil, errors.Wrapf(err, "Unable to create PVC for volume %v", *vol)
 		}
-		pv, err := kubevolume.CreatePV(ctx, cli, vol, vol.Type, annotations, nil, nil)
+
+		pvAnnotations := addPVProvisionedByAnnotation(nil, provider)
```

Comment:
Do we need the `annotations` from above to be passed in here? Does `kubevolume.CreatePVC` add values to the annotations map?

Reply:
`kubevolume.CreatePVC` doesn't add values to the annotations. But I think its better idea to have separate annotation values for PV and PVC.

Reply:
and the reason is, as we know these two resources can  have different set of labels.